src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.cs

```diff
@@ -880,8 +880,9 @@ public async Task SendAsync_ReadFromSlowStreamingServer_PartialDataReturned()
 
         [Theory]
         [InlineData(true)]
-        [InlineData(false)]
-        [InlineData(null)]
+        // [ActiveIssue("https://github.com/dotnet/runtime/issues/53087")]
+        // [InlineData(false)]
+        // [InlineData(null)]
```

Comment:
```suggestion
        [ActiveIssue("https://github.com/dotnet/runtime/issues/53087")]
        [InlineData(false)]
        [InlineData(null)]
```

Reply:
Maybe it's better to add that as a comment to prevent further questions... like
```suggestion
        // The following inputs are failing, see https://github.com/dotnet/runtime/issues/53087
        // [InlineData(false)]
        // [InlineData(null)]
```

Reply:
I actually moved it inside the code. I realizes that 2 tests are defined on an abstract base and I was suppressing the tests for all HTTP versions. So placed `if` in code and left the active issue as a comment, we have such approach in many other places in code.